org.debian.TuxPuck.json

```diff
@@ -0,0 +1,102 @@
+{
+    "app-id": "org.debian.TuxPuck",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "tuxpuck",
+    "rename-appdata-file": "tuxpuck.appdata.xml",
+    "rename-desktop-file": "tuxpuck.desktop",
+    "rename-icon": "tuxpuck",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/aclocal",
+        "/share/man",
+        "*.la", "*.a"
+    ],
+    "modules": [
+        {
+            "name": "SDL",
+            "rm-configure": true,
+            "config-opts": ["--disable-static"],
+            "cleanup": [
+                "/bin"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.libsdl.org/release/SDL-1.2.15.tar.gz",
+                    "sha256": "d6d316a793e5e348155f0dd93b979798933fb98aa1edebcc108829d6474aad00"
+                },
+                {
+                    "type": "patch",
+                    "path": "sdl-libx11-build.patch"
+                },
```

Comment:
Lots of things seems to use SDL1, and this even has a patch. I wonder if we should add it to shared-modules.